### PR TITLE
ADAPT-3702: Add intro to ankle region

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Add intro (richtext) to ankle region
- Refactor ankle.js to separate out styles

# Review By (Date)
- before end of this sprint

# Criticality
- 6/10

# Review Tasks

## Setup tasks and/or behavior to test

Note: Had some comms with Bhavika and Kerri and they would like to tighten up the spacing between ankle title and intro compared to the current mockup. This PR has taken into account that change which makes the netlify preview slightly different from the Figma mocks. Also, going with the icon card width in SAA Homesite when there are only 2 cards, so that's also slightly different from the mockup.
1. Go to https://deploy-preview-184--stanford-alumni-uat.netlify.app/travel-study/curated-collections/
2. This page uses the `Dark Background Page` type. Look at the ankle and check that an intro could be added below the heading
3. Go to https://deploy-preview-184--stanford-alumni-uat.netlify.app/test/yvonne/test-basic-page-preset/
4. This page uses the Basic Page page type. Check that the ankle could set up to be the same as above
5. Go to https://deploy-preview-184--stanford-alumni-uat.netlify.app/events/
6. Check that for the SAA homesite pages, the ankle stays the same as before.

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: src/components/partials/ankle/ankle.jsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            <div className={styles.introWrapper}>
+              <RichTextRenderer
+                wysiwyg={ankleIntro}
+                className={styles.intro}
+                isDark={isAnkleDark}
+              />
+            </div>
+          )}
+        </Container>
+      )}
```
Comment (yvonnetangsu) [inline src/components/partials/ankle/ankle.jsx:88]: I renamed the file to jsx so it makes it look like the whole file is new, but only this part has significant change. Conditionally render intro here, and adjust spacing on the Heading depending on whether there is an intro

File: src/components/partials/ankle/ankleStyles.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const root = 'ankle su-relative su-rs-py-7';
+export const heading = 'su-mt-[-0.6em] su-mx-auto su-max-w-900';
+export const headingMarginNoIntro = 'su-rs-mb-3';
+export const headingMarginHasIntro = 'su-mb-06em';
+export const introWrapper =
+  'su-big-paragraph su-max-w-prose su-mx-auto su-text-center su-rs-mb-3';
+export const intro = 'children:su-leading-display';
+export const iconCardGrid =
+  'su-gap-y-2xl md:su-gap-y-[80px] xl:su-gap-y-[100px]';
```
Comment (yvonnetangsu) [inline src/components/partials/ankle/ankleStyles.js:9]: Trying out the separation of style and markup 😄

File: tailwind.config.js [hunk 3]
```diff
@@ -2,90 +2,87 @@
...
-    screens: require(decanter + "/theme/screens.js")(),
-    spacing: require(decanter + "/theme/spacing.js")(),
-    transitionDuration: require(decanter + "/theme/transitionDuration.js")(),
-    width: require(decanter + "/theme/width.js")(),
+    borderRadius: require(decanter + '/theme/borderRadius.js')(),
+    borderWidth: require(decanter + '/theme/borderWidth.js')(),
+    colors: require(decanter + '/theme/colors.js')(),
+    fontFamily: require(decanter + '/theme/fontFamily.js')(),
+    fontSize: require(decanter + '/theme/fontSize.js')(),
+    fontWeight: require(decanter + '/theme/fontWeight.js')(),
```
Comment (yvonnetangsu) [inline tailwind.config.js:26]: Nothing to see here. Just linting changing double quotes to singles.

**Comment by yvonnetangsu** [comment]
@mattanglin Ready for you to look at. I haven't added the ankle region to the Trip and Trip Filter pages yet, but have added the schemas to those pages in Storyblok.

**Review by mattanglin — APPROVED** [review]
lgtm 👍🏼

**Pull request merged**